From 23b8862eccaaf3694664b1e1086dc3da8ca28a2f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 27 Dec 2017 22:26:44 +0800
Subject: [PATCH] MLK-17314-3 arm: dts: imx7ulp: update nmi irq number

On i.MX7ULP B0 chip, NMI irq number is changed,
update it to make VLLS/VLPS work.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index a558a13e..7d6507f 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -187,7 +187,7 @@
 		nmi: nmi@40220000 {
 			compatible = "fsl,imx7ulp-nmi";
 			reg = <0x40220000 0x1000>;
-			interrupts = <GIC_SPI 74 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts = <GIC_SPI 90 IRQ_TYPE_LEVEL_HIGH>;
 			status = "okay";
 		};
 
-- 
2.9.2

